packages/api/src/command/medical/cq-directory/get-cq-directory-entry.ts

```diff
@@ -18,3 +18,20 @@ export const getCQDirectoryEntryOrFail = async ({
   if (!organization) throw new NotFoundError(`Could not find CQ organization`, undefined, { oid });
   return organization;
 };
+
+export const getCQDirectoryEntriesIdsAndLastUpdated = async (
+  oids: string[]
+): Promise<{ id: string; oid: string; lastUpdated: string }[]> => {
```

Comment:
maybe make this a return type like:

`interface CQDirectoryEntryIdsAndLastUpdated {
  id: string;
  oid: string;
  lastUpdated: string;
}`